api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -52,6 +67,19 @@ message IPTagging {
     repeated config.core.v3.CidrRange ip_list = 2;
   }
 
+  // Optional header to use for ip-tagging instead of using
+  // default header ``x-envoy-ip-tags``.
```

Comment:
Again if sanitization is different for CustomHeader than DefaultHeader let's call out here.  "This header will be sanitized based on the config in ``ip_tag_custom_header_action`` rather than the defaults for x-envoy prefixed headers